components/common/map/EnhancedLocationPicker.tsx

```
@@ -1,27 +1,15 @@
-  Divider,
-  Button,
-  message,
-} from 'antd';
-import {
-  EnvironmentOutlined,
-  RadiusUprightOutlined,
-  SearchOutlined,
-  AimOutlined,
-} from '@ant-design/icons';
+
+import { Card, InputNumber, Slider, Space, Typography, Divider, Button, message, Input } from 'antd';
```

ESLint

[error] Replace `·Card,·InputNumber,·Slider,·Space,·Typography,·Divider,·Button,·message,·Input·` with `⏎··Card,⏎··InputNumber,⏎··Slider,⏎··Space,⏎··Typography,⏎··Divider,⏎··Button,⏎··message,⏎··Input,⏎`

(prettier/prettier)